db/schema.rb

```diff
@@ -24,7 +24,7 @@
   end
 
   create_table "conditions", force: :cascade do |t|
-    t.date "date"
```

Comment:
We think this won't be an issue and plausibly resolved by dropping and remigrating.